Doc/library/os.path.rst

```diff
@@ -485,8 +485,17 @@ the :mod:`glob` module.)
 
    Split the pathname *path* into a pair ``(root, ext)``  such that ``root + ext ==
    path``, and *ext* is empty or begins with a period and contains at most one
-   period. Leading periods on the basename are  ignored; ``splitext('.cshrc')``
-   returns  ``('.cshrc', '')``.
+   period.
+
+   If the path contains no extension, ``ext`` will be ``''``.
+   e.g. ``splitext('bar')`` returns ``('bar', '')``
+
+   If the path contains an extension, then ``ext`` will be set to this extension,
+   including the leading period. Note that previous periods will be ignored.
+   e.g. ``splitext('foo.bar.exe')`` returns ``('foo.bar', '.exe')``.
```

Comment:
What you've done absolutely does accurately follow the style and patterns seen in this documentation thus far. I think it would be nice to do repr examples like this, though:

```rst
If the path contains no extension, *ext* will be an empty string::

    >>> splitpath('bar')
    ('bar', '')

If the path contains an extension, then *ext* will be set to this extension,
including the leading period. Note that previous periods will be ignored::

    >>> splittext('foo.bar.exe')
    ('foo.bar', 'ext')

Leading periods on the basename are also ignored::

    >>> splitext('.cshrc')
    ('.cshrc', '')
```

BUT

1. That's just me
2. It breaks from the style of other in lined examples throughout this doc

So, I'd recommend just wait and see if / how others react to this comment, and only go ahead and restructure it like this if others agree with me, because might not!